src/main/java/io/github/mzmine/modules/identification/ms/localdatabasesearch/LocalDatabaseSearchModule.java

```diff
@@ -85,7 +97,73 @@ public void runModule(@Nonnull MZmineProject project,
             return;
         }
 
-        System.out.println("Start identification using local database");
+        // Convert the input annotations into ionAnnotations
+        List<IonAnnotation> ionAnnotations = new ArrayList<IonAnnotation>();
+        Scanner scanner = new Scanner(annotations);
+        while (scanner.hasNextLine()) {
+            String line = scanner.nextLine();
+            if (line != null) {
+
+                // Only process lines which start with an integer
+                char firstChar = line.charAt(0);
+                if (Character.digit(firstChar, 10) >= 0) {
+                    String[] lineArray = line.split(separator);
+
+                    String annotationId = lineArray[0];
+                    Double mz = Double.parseDouble(lineArray[1]);
+                    Float rt = Float.parseFloat(lineArray[2]) * 60;
+                    String name = lineArray[3];
+
+                    // If formula column is present then add chemical structure
+                    IAtomContainer chemicalStructure = null;
+                    if (lineArray.length > 4) {
+                        String formula = lineArray[4];
+                        // chemicalStructure
+                        /*
+                         * TODO
+                         */
+                    }
```

Comment:
@tomas-pluskal Can you help me with converting the formula (String) to the IAtomContainer?